python/python/lancedb/query.py

```diff
@@ -1189,18 +1190,51 @@ def _combine_hybrid_results(
             fts_results = LanceHybridQueryBuilder._rank(fts_results, "_score")
 
         # normalize the scores to be between 0 and 1, 0 being most relevant
-        vector_results = LanceHybridQueryBuilder._normalize_scores(
-            vector_results, "_distance"
-        )
+
+        if vector_results.num_rows > 0:
```

Comment:
Why the `num_rows > 0` check?  Just to avoid doing work if we don't have to?  Will that mean that `original_distances` is not defined when it's referenced down below?

Reply:
Oh I should leave a comment there about this. I found that sometimes `vector_results` or `fts_results` are empty, and in that case they turn out to be missing the `_rowid` column. So we skip the normalization step in that case.